### DISCUSSION TITLE
Is there a guide/docs to redesign the Admin UI

### DISCUSSION THREAD

**Discussion by annardini** [payloadcms/payload#6328] [Q&A]
I think it would be a good idea to give developers the option to completely create a new UI for payload this is much easier with the move to Next JS as we can just remove the route(s) under `app/(payload)/admin` but there is no documentation that gives a better understanding on how to go about this as payload will redirect to different URLS which would provide 404's if not caught which is bad user experience. 

I would like to redesign the admin UI using Shadcn which I like the look of. However, trying to add tailwind to the payload admin ui breaks a lot of other styles and passing components and correcting those errors will make the project messy.

Does anyone have any suggestions?

**Comment by datnt19213** [comment]
Nah, ... i want the same things that can custom all UI of /admin payload main and CRUD UI screen, all of them. But i confuse how to do that, if replace class with css or sth like this i think it may be possible but use tailwind instead of css/scss is really difficult. Someone have the ways to resolve them? 🥲😭
